src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -843,6 +854,22 @@ public function morphTo($name = null, $type = null, $id = null)
     }
 
     /**
+     * Fetch the class name from the morph map, or return the original if it's not defined
```

Comment:
The short description must be no longer than 80 chars including the 5 spaces before the *, the *, and the space after it, and must end in a full stop.